save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/ExecutionLabels.kt

```diff
@@ -96,7 +96,8 @@ internal class ExecutionStatisticsValues(executionDto: ExecutionDto?) {
         this.failedTests = executionDto?.failedTests?.toString() ?: "0"
         this.runningTests = executionDto?.runningTests?.toString() ?: "0"
         this.passRate = executionDto
-            ?.let { "${calculateRate(it.passedTests, it.allTests)}" }
+            ?.let { calculateRate(it.passedTests, it.allTests) }
+            ?.toString()
```

Comment:
When have we broken it?

Reply:
I guess it was done in such way initially.
`allTests == 0` is wear case